MoabTests/Models.Helpers/ExerciseImportHelperShould.cs

```diff
@@ -55,5 +68,123 @@ public ExerciseImportHelperShould()
         }
 
         // Add Unit Tests here...
+        [Fact]
+        public void HaveFourExercises()
+        {
+            // Arrange
+            // Act
+            var count = _existingExercises.Count;
+            // Assert
+
+            count.Should().Be(4);
+        }
+
+        [Fact]
+        // Checks whether the LineSplitting Function works
+        public void LineSplitting1()
+        {
+            // Arrange
+            ExerciseImportHelper TestImport = new ExerciseImportHelper();
+            // Act
+            string[] ex1Split = new string[14];
+            ex1Split = TestImport.TestLineSplit(CSVInputex1);
+            string ex1test = ex1Split[0] + ',';
+            for (int i = 1; i < 14; i++)
+            {
+                ex1test += (ex1Split[i] + ',');
+            }
+            // Assert
+            CSVInputex1.Should().Equals(ex1test);
+        }
+
+        [Fact]
+        public void TestSplitting2()
+        {
+            // Arrange
+            ExerciseImportHelper TestImport = new ExerciseImportHelper();
+            // Act
+            string[] ex2Split = new string[14];
+            string ex2test = ex2Split[0] + ',';
+            for (int i = 1; i < 14; i++)
+            {
+                ex2test += (ex2Split[i] + ',');
+            }
+            // Assert
+            CSVInputex2.Should().Equals(ex2test);
+        }
+
+        [Fact]
+        public void TestSplitting3()
+        {
+            // Arrange
+            ExerciseImportHelper TestImport = new ExerciseImportHelper();
+            // Act
+            string[] ex3Split = new string[14];
+            string ex3test = ex3Split[0] + ',';
+            for (int i = 1; i < 14; i++)
+            {
+                ex3test += (ex3Split[i] + ',');
+            }
+            // Assert
+            CSVInputex3.Should().Equals(ex3test);
+        }
+
+        [Fact]
+        public void TestSplitting4()
+        {
+            // Arrange
+            ExerciseImportHelper TestImport = new ExerciseImportHelper();
+            // Act
+            string[] ex4Split = new string[14];
+            string ex4test = ex4Split[0] + ',';
+            for (int i = 1; i < 14; i++)
+            {
+                ex4test += (ex4Split[i] + ',');
+            }
+            // Assert
+            CSVInputex4.Should().Equals(ex4test);
+        }
+
+        [Fact]
+        // Checks whether the LineSplitting Function works
+        public void ImportTesting()
```

Comment:
Think about the name of the function so it makes sense with the class name.

Reply:
Done